opentelemetry/deployment/otelcol.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: otel-collector-example
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+  namespace: otel-collector-example
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    config.openshift.io/inject-trusted-cabundle: "true"
+  name: trusted-ca
+  namespace: otel-collector-example
+data: {}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+  namespace: otel-collector-example
+rules:
+- apiGroups: [""]
+  resources: ["pods", "namespaces", "infrastructures"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-deployment
+  namespace: otel-collector-example
+roleRef:
+  kind: ClusterRole
+  name: otel-collector
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: otel-collector-example
+spec:
+  mode: deployment
+  serviceAccount: otel-collector-deployment
+  volumes:
+    - name: trusted-ca
+      configMap:
+        name: trusted-ca
+        items:
+          - key: ca-bundle.crt
+            path: tls-ca-bundle.pem
+  volumeMounts:
+    - name: trusted-ca
+      mountPath: /etc/pki/ca-trust/extracted/pem
+      readOnly: true
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+      opencensus:
+      otlp:
+        protocols:
+          grpc:
+          http:
+      zipkin:
+    processors:
+      k8sattributes:
+      resourcedetection/env:
+        detectors: [env]
+        timeout: 2s
+      resourcedetection/openshift:
+        detectors: [openshift]
+        timeout: 2s
+        openshift:
+          tls:
+            insecure: false
+            ca_file: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+    exporters:
+      otlp:
+        endpoint: "tempo-simplest-distributor.svc:4317"
+        tls:
+          ca_file: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
```

Comment:
Are you sure the distributor is secured with CA cert? cc) @rubenvp8510 

As far as I know the CA is needed only when collector reports data to gateway.

Reply:
So... should I remove this?

Reply:
If the ingester is not using TLS then yes. 

I assume you have tested the e2e setup.

Reply:
I'll test it by removing this. I tested with this value set and everything worked